doc/release-notes-26201.md

```diff
@@ -0,0 +1,5 @@
+RPC
+---
+
+- 'taproot' has been removed from 'getdeploymentinfo', rather than marked as buried,
```

Comment:
```suggestion
- 'taproot' has been removed from 'getdeploymentinfo',
```

nit: I think release notes should describe changes, not provide extended discussion about alternatives